# Biotagging

Here we provide additional information such as scripts and data for the Biotagging Project.


## Project Description

Interrogation of gene regulatory circuits in complex organisms requires precise
tools for the selection of individual cell types, as well as robust methods for tissue- specific labeling and biochemical profiling of target proteins. By exploiting multiple transgenesis strategies, we have developed a tissue-specific binary in vivo biotinylation system in zebrafish termed “biotagging”, a versatile methodology that uses genetically- encoded components to biotinylate target proteins, enabling in-depth genome-wide analyses of their molecular interactions. Using tissue-specific transgenic drivers and individual cell compartment effector lines from our “biotagging” toolkit, we demonstrate the specificity of our approach at the biochemical, cellular and transcriptional level. By characterizing the *in vivo* transcriptional landscape of migratory neural crest and myocardial cells in two different cellular compartments (ribosomes and nucleus), we identify a comprehensive network of protein-coding and non-coding RNAs and uncover cis-regulatory modules and regulatory logic conferring cell-specific identity, embedded in the complexity of the non-coding nuclear transcriptomes. Our study demonstrates that “biotagging” eliminates background inherent to complex embryonic environments and allows analyses of molecular interactions in any cellular context at highest resolution.


## Highlights

* Biotagging is a binary genetic system for tissue-specific, subcellular *in vivo* biotinylation in zebrafish.
* Biotagging enables comprehensive transcriptional pathway analyses of migrating neural crest and myocardial cells
* Strand-specific nuclear RNA profiling reveals tissue-specific bi-directional transcription across putative cis-regulatory elements and uncovers tight developmental regulation of a variety of non-coding RNA species.
* Molecular logic for cell identity is encrypted in the nuclear transcriptome, as revealed by analysing the migratory neural crest.

## Keywords

zebrafish, neural crest, *in vivo* biotinylation of nuclei, *in vivo* biotinylation of polysomes, lincRNA, RNA-seq, ATAC-seq, transribed enhancers

## Research Organisms

*Danio rerio*

# Manuscript and contact information
The Biotagging toolkit for analysis of specific cell populations in zebrafish reveals gene regulatory logic encoded in the nuclear transcriptome
Le A. Trinh *<sup>1</sup>, Vanessa M. R. Chong* <sup>2</sup>, Daria Gavriouchkina<sup>2</sup>, Tatiana Hochgreb-Hagele<sup>2,3</sup>, Upeka Senanayake<sup>2</sup>, Scott E. Fraser<sup>1</sup>, Tatjana Sauka-Spengler<sup>2</sup>


**\*Corresponding author: Tatjana Sauka-Spengler (tatjana.sauka-spengler@imm.ox.ac.uk)**

**Vanessa M. R. Chong, Daria Gavriouchkina, Tatiana Hochgreb-Hagele, Upeka Senanayake,Tatjana Sauka-Spengler**: [Sauka-Spengler lab](http://www.lgm.upmc.fr/mcl/index.html)

**Le A. Trinh, Scott E. Fraser**: [Fraser lab](https://www.iamphioxus.org)

**Tatiana Hochgreb-Hagele**: [Bronner lab](http://cmb.gu.se/english/about_us/staff?languageId=100001&userId=xwarrj)

## Affliations
1. University of Southern California
Molecular and Computational Biology
Los Angeles, CA 90089
2. University of Oxford
Weatherall Institute of Molecular Medicine
Radcliffe Department of Medicine
Oxford, OX3 9DS
3. California Institute of Technology
Division of Biology and Biological Engineering
Pasadena, CA 91125

![](other/affiliations.png)

## Funding

![](other/funding.png)
